Apps/Examples/Examples/All Examples/ViewportExample.swift

```diff
@@ -36,11 +36,11 @@ final class ViewportExample: UIViewController, ExampleProtocol {
         mapView.mapboxMap.setCamera(to: CameraOptions(center: cupertino, zoom: 14))
 
         mapView.location.options.puckType = .puck2D(.makeDefault(showBearing: true))
```

Comment:
Could we add a quick comment about what these different options mean (i.e. .course vs. .heading)? Or maybe point to the documentation.

Reply:
will update docs :)